Core/Filesystem/nand_core.c

```diff
@@ -0,0 +1,366 @@
+/*
+ * Copyright (C) 2022  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/*
+ * nand_core.c
+ *
+ *  Created on: Jun 14, 2022
+ *      Author: Robert Taylor
+ */\
+#include <stdint.h>
```

Comment:
In interest of lightweight software, are these required? trying to keep required packages to a minimum

Reply:
I don't think stdint adds any code volume. I'll see about string

Reply:
Linker will optimize out and string functions not used, no need to exclude it